system_tests/system_tests_sync/test_external_accounts.py

```diff
@@ -100,6 +102,23 @@ def service_account_info(service_account_file):
         yield json.load(f)
 
 
+@pytest.fixture
+def aws_credentials(service_account_file, service_account_info, authenticated_request):
+    credentials = service_account.Credentials.from_service_account_file(
+        service_account_file, scopes=["https://www.googleapis.com/auth/cloud-platform"]
+    )
+    result = authenticated_request(credentials)(
+        url="https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/{}:generateIdToken".format(
+            service_account_info["client_email"]
+        ),
+        method="POST",
+        body=json.dumps({"audience": "104692443208068386138", "includeEmail": True}),
```

Comment:
I believe `104692443208068386138` is `service_account_info["client_id"]`

Reply:
Yep.  Done